src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -225,11 +225,6 @@ void DefaultDACVerifier::VerifyAttestationInformation(const DeviceAttestationVer
         VerifyOrExit(!paaVidPid.mProductId.HasValue(), attestationError = AttestationVerificationResult::kPaaFormatInvalid);
     }
 
-#if !defined(CURRENT_TIME_NOT_IMPLEMENTED)
-    VerifyOrExit(IsCertificateValidAtCurrentTime(info.dacDerBuffer) == CHIP_NO_ERROR,
-                 attestationError = AttestationVerificationResult::kDacExpired);
-#endif
-
```

Comment:
I had read the spec as still requiring this:

```
Given a list of DER-encoded certificates in RFC 5280 format, starting at the end-entity (leaf)
certificate, and following the chain of trust towards the root, Crypto_VerifyChainDER() performs all
the necessary validity checks on certificates.
The Validity period validation for the root and optional intermediate certificates is performed
against the notBefore timestamp of the end-entity (leaf certificate) used as value for the current
time.
```

Only PAA and PAI seem to be explicitly specified has having different rules for Validity period validation.

Reply:
Analyzing this section of the spec I agree with you that the DAC must be valid at the current time. So I am returning that check. And then I also agree with your second statement below that the DAC NotAfter should be set to X509NoWellDefinedExpirationDate in order to achieve non-expiration for the chain.
The spec should be updated to require one of the following:
  - require non-expiration for the DAC
  - or don't check validity of the DAC